back/engines/commercial/analytics/spec/models/analytics/fact_participation_spec.rb

```diff
@@ -92,5 +95,33 @@
         expect(described_class.find(input.id).dimension_type.name).to eq('survey')
       end
     end
+
+    context 'when a basket is created' do
+      let(:basket) { create(:basket) }
+
+      it 'is also available as a participation fact' do
+        described_class.find(basket.id)
+        expect(described_class.find(basket.id).dimension_type.name).to eq('basket')
+      end
+    end
+
+    context 'when an event is attended' do
+      let(:event_attendance) { create(:event_attendance) }
+
+      it 'is also available as a participation fact' do
+        described_class.find(event_attendance.id)
+        expect(described_class.find(event_attendance.id).dimension_type.name).to eq('event_attendance')
+      end
+    end
+
+    context 'when a follower is created' do
+      let(:follower) { create(:follower) }
+
+      it 'is also available as a participation fact' do
+        described_class.find(follower.id)
+        expect(described_class.find(follower.id).dimension_type.name).to eq('follower')
+        expect(described_class.find(follower.id).dimension_type.parent).to eq('project')
```

Comment:
Let's save a few db queries:
```suggestion
        participation = described_class.find(follower.id)
        expect(participation.dimension_type.name).to eq('follower')
        expect(participation.dimension_type.parent).to eq('project')
```